rbac/api/cross_access/view.py

```diff
@@ -107,6 +116,54 @@ def list(self, request, *args, **kwargs):
             return self.replace_user_id_with_info(result)
         return result
 
+    def partial_update(self, request, *args, **kwargs):
+        """Patch a cross-account request."""
+        validate_uuid(kwargs.get("pk"), "cross-account request uuid validation")
+        payload = request.data
+        for field in payload:
+            if field not in VALID_PATCH_FIELDS:
+                key = "cross-account request"
+                message = f"Field '{field}' is not supported. Please use one or more of: {VALID_PATCH_FIELDS}."
+                error = {key: [_(message)]}
+                raise serializers.ValidationError(error)
+
+        with tenant_context(Tenant.objects.get(schema_name="public")):
+            current = self.get_object()
```

Comment:
From my understanding, this will return the object based on the pk?

Reply:
That's my understanding too, yep!